include/benchmark/benchmark_api.h

```diff
@@ -674,8 +721,12 @@ class Fixture: public internal::Benchmark {
       this->TearDown(st);
     }
 
+    // These will be deprecated ...
```

Comment:
This change should be made as a separate PR, since it's unrelated to this patch AFAIK.